packages/legacy-workbench/src/js/angular/ttyg/services/agents.mapper.js

```diff
@@ -172,6 +174,7 @@ export const agentModelMapper = (data, localRepositoryIds) => {
         // Sets the isRepositoryDeleted flag based on whether the repository ID is not in the list of repositories.
         isRepositoryDeleted: !localRepositoryIds.includes(data.repositoryId),
         model: data.model,
+        contextSize: data.api === 'openai-assistants' ? null : data.contextSize,
```

Comment:
IMO, it would be better to add an "api" field to the model and set it here. Then, instead of using:
`$scope.showContextSize = ($scope.agentFormModel.contextSize !== null);` which isn’t very descriptive, consider using something like:
$scope.showContextSize = $scope.agentFormModel.api === ApiName.OPENAI_ASSISTANTS;
The names are just examples; they can be something like agentApiName, etc.